mirrord/operator/src/crd/policy.rs

```diff
@@ -51,6 +53,12 @@ pub struct MirrordPolicySpec {
     // TODO: make the k8s list type be set/map to prevent duplicates.
     /// List of features and operations blocked by this policy.
     pub block: Vec<BlockedFeature>,
+
+    /// List of environment variables that should be excluded when using mirrord.
+    ///
+    /// These environment variables won't be retrieved from the target even if the user
+    /// specifies them in their `feature.env.include` mirrord config.
+    pub env_vars_exclude: Option<HashSet<String>>,
```

Comment:
Should we go for literal keys or key regexes? :thinking:

Reply:
I thought we wanted regexes just like the env vars exclude filter thingy? I don't see why we would not want that?

Reply:
Changed the docs.